site2/docs/functions-develop.md

```diff
@@ -35,6 +35,20 @@ def process(input):
 ```
 For complete code, see [here](https://github.com/apache/pulsar/blob/master/pulsar-functions/python-examples/native_exclamation_function.py).
 
+Note that functions can be written in python2 or python3, but Pulsar
```

Comment:
Accepted, except I prefer "Your system will fail if you subsequently install any other package that depends on Python 2.x" (since by definition, if you've already installed a package which depends on python 2.x, you will already have "python" and Pulsar will use it)

Reply:
Yes, if so, you can use "Your system will fail if you subsequently install any other package that depends on Python 2.x".